Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string name = StringPrompt("Please introduce a name:");
```

Comment:
2nd person is missing;
Your demo could also call another method: PromptPersonBmi()